peer.go

```diff
@@ -10,9 +10,11 @@ import (
 	"sync"
 	"time"
 	"unicode"
+	"os"
 
 	"github.com/creack/pty"
 	"github.com/pion/webrtc/v3"
+    "github.com/riywo/loginshell"
```

Comment:
the indentation looks wrong. What IDE are using? I would suggest [vim-go](https://youtu.be/7BqJ8dzygtU) to prevent this kind of foolishness

Reply:
Sorry, using vscode, which is typically smart about this, if you forget to start a new line yourself and not paste the whole line.
Just pushed a minor spaces->tab fix directly in tuzig/master.